pkg/gorilla/gorilla.go

```diff
@@ -103,6 +103,16 @@ func (ws *WebSocket) Close() error {
 	return ws.Conn.WriteMessage(gorilla.CloseMessage, gorilla.FormatCloseMessage(CloseMessageCode, ""))
 }
 
+func (ws *WebSocket) LiveNotifications(id string) (chan rpc.RPCResponse, error) {
+	c, err := ws.createResponseChannel(id)
+	if err != nil {
+		ws.logger.Logger.Err(err)
+		ws.logger.LogChannel <- err.Error()
+		return nil, err
+	}
+	return c, nil
```

Comment:
we can make for make it shorter
```suggestion
	return c, err
```